include/abb_librws/rws_interface.h

```diff
@@ -204,17 +260,24 @@ class RWSInterface
   /**
    * \brief A method for collecting runtime information of the robot controller.
    *
-   * \return RuntimeInfo container for the runtime information.
+   * \return RuntimeInfo containing the runtime information.
    */
   RuntimeInfo collectRuntimeInfo();
   
   /**
    * \brief A method for collecting static information (at least during runtime) of the robot controller.
    *
-   * \return StaticInfo container for the static information (at least during runtime).
+   * \return StaticInfo containing the static information (at least during runtime).
    */
   StaticInfo collectStaticInfo();
-  
+
+  /**
+   * \brief A method for retrieving the RobotWare options present in the active robot controller system.
+   *
+   * \return std::vector<OptionInfo> containing a list of the present RobotWare options.
```

Comment:
Here also: `std::vector<OptionInfo>` is returned, "a list of the .." (ie: I'd not make this active).

Reply:
Ok.

Reply:
I will consider this in a future PR.